<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>logWindow</class>
 <widget class="QMainWindow" name="logWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>日志</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QMainWindow {
    border-radius: 12px;    /* 圆角半径 */
}
   </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">#centralwidget{
border-image: url(:/images/mainwindow/mainwindow_background.png) 0 0 0 0 stretch stretch;
border-radius: 12px;
}</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="titleBar" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>35</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>35</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">/* 关闭 */
#closeButton                     { image:url(&quot;:/images/mainwindow/close_normal.png&quot;); }
#closeButton:hover               { image:url(&quot;:/images/mainwindow/close_hover.png&quot;);  }

/* 最小化 */
#minButton                       { image:url(&quot;:/images/mainwindow/min_normal.png&quot;);   }
#minButton:hover                 { image:url(&quot;:/images/mainwindow/min_hover.png&quot;);    }

/* 最大化 */
#maxButton                       { image:url(&quot;:/images/mainwindow/max_normal.png&quot;);   }
#maxButton:hover                 { image:url(&quot;:/images/mainwindow/max_hover.png&quot;);    }

/* 让 titleBar 背景图铺满并自动拉伸 */
#titleBar {
    border-image: url(:/images/mainwindow/bar_background.png) 0 0 0 0 stretch stretch;
    border-top-left-radius:12px; border-top-right-radius:12px;
}

/* 通用尺寸&amp;去边框 */
#titleBar QToolButton {
    border: none;
    background: transparent;
}</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>8</number>
       </property>
       <property name="leftMargin">
        <number>20</number>
       </property>
       <property name="topMargin">
        <number>9</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>9</number>
       </property>
       <item>
        <widget class="QWidget" name="btnZone" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>8</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QToolButton" name="closeButton">
            <property name="minimumSize">
             <size>
              <width>15</width>
              <height>15</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>15</width>
              <height>15</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="minButton">
            <property name="minimumSize">
             <size>
              <width>15</width>
              <height>15</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>15</width>
              <height>15</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="maxButton">
            <property name="minimumSize">
             <size>
              <width>15</width>
              <height>15</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>15</width>
              <height>15</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>1279</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="contentWidget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="layout">
         <property name="spacing">
          <number>20</number>
         </property>
         <property name="leftMargin">
          <number>20</number>
         </property>
         <property name="topMargin">
          <number>20</number>
         </property>
         <property name="rightMargin">
          <number>20</number>
         </property>
         <property name="bottomMargin">
          <number>20</number>
         </property>
         <item>
          <widget class="QWidget" name="widget" native="true">
           <property name="styleSheet">
            <string notr="true">

#widget {
border-image: url(:/images/sideBar/black1.png) 0 0 0 0 stretch stretch;
border-radius: 12px;
}
</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>12</number>
            </property>
            <property name="topMargin">
             <number>12</number>
            </property>
            <property name="rightMargin">
             <number>12</number>
            </property>
            <property name="bottomMargin">
             <number>12</number>
            </property>
            <item>
             <widget class="QPlainTextEdit" name="plainTextEdit">
              <property name="styleSheet">
               <string notr="true">
#plainTextEdit{
border-image: url(:/images/sideBar/black1.png) 0 0 0 0 stretch stretch;
color:rgb(55,55,55);
}
</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
